datalad/__init__.py

```diff
@@ -106,5 +107,3 @@ def teardown_package():
     lgr.debug("Teardown tests. " + msg)
     for path in _TEMP_PATHS_GENERATED:
         rmtemp(path, ignore_errors=True)
-
-lgr.log(5, "Done importing main __init__")
```

Comment:
This was an attention test. I can assert that you are performing on (at least) average level. Congrats!